### PR TITLE
Add version adornments.

### PR DESCRIPTION
**Pull request by LeoTindall**
Relevant to #142.

The file "common/version.js" should be updated in the LAST commit before a tag is made for that version. 

The contents of the created version object are displayed on the app and are available as JSON in the API, from /api/version, in case we ever want to make mobile apps or support very long-running displays (e.g. the "wall graph" use case).

### CODE REVIEW

File: src/client/app/components/HeaderComponent.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { Link } from 'react-router';
 import { Button } from 'react-bootstrap';
 import LogoComponent from './LogoComponent';
 import UIModalComponent from './UIModalComponent';
+import VERSION from '../../../common/version';
```
Comment (johndiiorio) [inline src/client/app/components/HeaderComponent.jsx]: Why is this being important in the HeaderComponent?

File: src/common/version.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	this.major = versionParts[0];
+	this.minor = versionParts[1];
+	this.patch = versionParts[2];
+}
+
+/**
+ * @returns {string} a string representation of the Semver version of the application,
+ * in the form v1.2.3
+ */
+OEDVersion.prototype.toString = function toString() {
```
Comment (johndiiorio) [inline src/common/version.js]: What is the point of parsing the package.json, splitting the major, minor, patch, then recombining them? Why not just prepend 'v' to the beginning of the string?

File: src/server/app.js [hunk 3]
```diff
@@ -29,6 +30,13 @@ app.use('/api/readings', readings);
 app.use('/api/login', login);
 app.use('/api/verification', verification);
 
+/**
+ * Returns the version of the application to the client.
+ */
+app.get('/api/version', (req, res) => {
```
Comment (johndiiorio) [inline src/server/app.js]: Even though this is really small, I would prefer this to be in its own route.

File: src/common/version.js (hunk 2 above)
Comment (LeoTindall) [reply src/common/version.js]: It has to be an object to return it as JSON.

**Pull request merged**